### ISSUE TITLE
Wibox isn't clickable after random amount of time

### ISSUE THREAD

**Issue by localhostport80** [awesomeWM/awesome#669]
I am using Awesome v3.5.8 with Archlinux and copycat-killer themes and very often, after a random amount of time, my statusbar isn't clickable anymore.

So changing the layout by clicking the corresponding icon, or calling the right click menu doesn't work anymore. Only with the keyboard shortcuts. For solving the problem i need to log out and in again.

**Comment by kapsh** [comment]
Do you mean wibox? I have similar problem with it: when I change layout from en to ru wibox stops responding to mouse clicks. 
Somewhere I found oneline workaround:
`xkbcomp $DISPLAY - | egrep -v 'group . = AltGr;' | xkbcomp - $DISPLAY &> /dev/null `
which temporary fix this.

**Comment by localhostport80** [comment]
Thx. I will give it a try as soon as it happens again. Hopefully we'll find a permanent solution.

**Comment by psychon** [comment]
@localhostport80 There might be a bug in Qt where it starts stealing mouse clicks after an input device hotplug (e.g. remove or connect a USB mouse). It can be recovered from by closing all programs using Qt (or upgrading Qt, but I'm not sure to which version). Could you also test this?

**Comment by localhostport80** [comment]
@psychon 5 minutes ago I had the same problem again. It was right after bootup. The only programs running, as they are in my rc.lua, are *octopi-notifier*, *owncloud*, *xflux* and the *xscreensaver* daemon. After I noticed this problem again, I closed the *owncloud* client and it worked again! It seems like the owncloud client is the origin of this issue.

@kapsh Another approach I chose was, taking a look whether there are errors returned, using this strategy http://awesome.naquadah.org/wiki/FAQ#Where_are_logs.2C_error_messages_or_something.3F There were some errors I could solve. Maybe it helps.

Nevertheless in some way I rely on this owncloud client, so I will keep an eye on this and tell you what I found :+1: .
